.circleci/config.yml

```diff
@@ -46,4 +46,4 @@ jobs:
     working_directory: /mnt/ramdisk
     steps:
       - setup
-      - run: npm test
+      - run: NODE_OPTIONS=--openssl-legacy-provider npm test
```

Comment:
This fixes an issue running the test, an example of such failing test can be found [here](https://app.circleci.com/pipelines/github/webpack-contrib/webpack-hot-middleware/105/workflows/7c103fb7-2c31-404f-bd46-739091d521ea/jobs/778) the adopted fix was suggested [here](https://github.com/webpack/webpack/issues/14532#issuecomment-947012063).